components/eventing-controller/reconciler/backend/reconciler.go

```diff
@@ -97,7 +97,6 @@ func (r *Reconciler) Reconcile(ctx context.Context, _ ctrl.Request) (ctrl.Result
 	}); err != nil {
 		return ctrl.Result{}, err
 	}
-	r.namedLogger().Debugw("found secrets with BEB label", "count", len(secretList.Items))
```

Comment:
Why are we removing this? It is a debug message and kind of useful.

Reply:
This logs creates a lot of noise even in debug mode. Instead, if we found more than one secret with the same label we'll print the found secrets count as part of the context (please check this [commit](https://github.com/kyma-project/kyma/pull/11651/commits/22784ae6b998fd677ecbeab39041f1d69c6bb29e)).